packages/edit-site/src/components/site-hub/index.js

```diff
@@ -41,20 +42,15 @@ const SiteHub = forwardRef( ( props, ref ) => {
 	const disableMotion = useReducedMotion();
 	const { setCanvasMode } = unlock( useDispatch( editSiteStore ) );
 	const { clearSelectedBlock } = useDispatch( blockEditorStore );
-	const isBackToDashboardButton = canvasMode === 'view';
-	const showLabels = canvasMode !== 'edit';
-	const siteIconButtonProps = isBackToDashboardButton
-		? {
-				href: dashboardLink || 'index.php',
-				label: __( 'Go back to the Dashboard' ),
-		  }
-		: {
-				label: __( 'Open Navigation Sidebar' ),
-				onClick: () => {
-					clearSelectedBlock();
-					setCanvasMode( 'view' );
-				},
-		  };
+	const siteIconButtonProps = {
+		label: __( 'Open Admin Sidebar' ),
+		onMouseDown: () => {
```

Comment:
I'm late to the party since this has already been merged, but is there a reason this was changed from `onClick` to `onMouseDown`? 
As the name implies, `onMouseDown` only works with the mouse (or other pointing devices), and that breaks keyboard navigation 🤔

Reply:
That's an oversight of one of the rebases — the intention was to allow triggering the layout switch faster before click release.